docs/sources/flow/reference/components/otelcol.receiver.vcenter.md

```diff
@@ -0,0 +1,129 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/otelcol.receiver.vcenter/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/otelcol.receiver.vcenter/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/otelcol.receiver.vcenter/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.receiver.vcenter/
+title: otelcol.receiver.vcenter
+description: Learn about otelcol.receiver.vcenter
+---
+
+# otelcol.receiver.vcenter
+
+`otelcol.receiver.vcenter` accepts telemetry data from a 
+vCenter or ESXi host running VMware vSphere APIs and
+forwards it to other `otelcol.*` components.
```

Comment:
I think this paragraph needs a few more things:
* We need to make it clear that this component only works with metrics.
* Which metrics are produced? Seems it is [these](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.87.0/receiver/vcenterreceiver/documentation.md) ones? If we are not going to list them in our docs, it would be good to link to the OTel page instead. However, I'd prefer if we don't link to the OTel page because it contains yaml config for the collector which our users will find confusing. Could you try to list them here please?
* Do we need to list the [prerequisites](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.87.0/receiver/vcenterreceiver/README.md#prerequisites)? They seem useful.

Reply:
Do we need to include everything that it's in the component readme? Or linking to it would provide users some understanding what are they using?

Reply:
> Do we need to include everything that it's in the component readme?

No - we might include more information, or less. We need to assume that the users will not read the Collector docs and that our doc page is the first page they are seeing for this component. 

> Or linking to it would provide users some understanding what are they using?

In general we avoid linking to OTel docs because they tend to contain Collector-specific yaml config which confuses the users. It's best to include all the relevant info in our own docs, unless there is a strong reason not to.